standalone-metastore/metastore-server/pom.xml

```diff
@@ -720,6 +720,14 @@
         <groupId>org.datanucleus</groupId>
         <artifactId>datanucleus-maven-plugin</artifactId>
         <version>5.2.1</version>
+        <!-- The plexus-contianer-default can be removed once we move to datanucleus-maven-plugin version >= 6.0.0-release-->
+        <dependencies>
+          <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-container-default</artifactId>
+            <version>${plexus.version}</version>
+          </dependency>
+        </dependencies>
```

Comment:
The same argument that holds for the shade plugin and `log4j-core:2.1` also applies here. The fact that datanucleus plugin has a dependency to log4j is not very problematic since it is not a Hive product/module dependency. log4j will only be used during the build and this usage is rather fine.

Nevertheless, if you are confident that upgrading the plexus `1.5.6` version is fully compatible with `5.2.1` I am OK to make this change although I don't find it necessary. I checked the enhanced classes before and after the change and they seem identical so I am ok to leave this as is.